tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Keramelizer.scala

```diff
@@ -64,6 +64,39 @@ class Keramelizer(gen: UniqueNameGenerator, tracker: TransformationTracker)
       tla
         .not(tla.in(x, setX) ? "b")
         .typed(Map("b" -> BoolT1()), "b")
+    // rewrite POWSET A \subseteq POWSET B
+    // into A \subseteq B
+    case OperEx(TlaSetOper.subseteq, OperEx(TlaSetOper.powerset, setX), OperEx(TlaSetOper.powerset, setY)) =>
+      transform(
+          tla
+            .subseteq(setX, setY)
+            .typed(Map("b" -> BoolT1()), "b")
```

Comment:
This is not something you'd be expected to know, but we've phased out the `.typed(map, key)` and `?` syntax for `as` in new code, e.g.
```suggestion
            .subseteq(setX, setY) as BoolT1()
```